querydsl-core/src/main/java/com/querydsl/core/group/GroupBy.java

```diff
@@ -16,10 +16,10 @@
 import java.math.MathContext;
 import java.util.*;
 
-import com.mysema.commons.lang.Pair;
 import com.querydsl.core.ResultTransformer;
 import com.querydsl.core.types.Expression;
 import com.querydsl.core.types.Projections;
+import com.querydsl.core.util.Pair;
```

Comment:
As demonstrated by this test, it can be expected users actually use Pair in their codebases, so we would have to make a new major release.

Reply:
Perhaps its better to shade the library for now or leave the classes under the same package. We know for a fact that its apache licensed anyway.

Reply:
We could use the same package name. My pair should be binary compatible with the old implementation.

Reply:
I think  I favor that change.